### DISCUSSION TITLE
multiple cursor support on tp.file.cursor() function

### DISCUSSION THREAD

**Discussion by tobulli** [SilentVoid13/Templater#148] [Ideas & Feature Requests]
Hi, really getting started with templater at the moment. Amazing work!
I would love to be able to use the multiple cursor option from obsidian within a template with the tp.file.cursor() function. So for example all function calls with the same order number are multiple cursors. Not sure if that is possible, but it would help for a lot of usecases.

**Comment by SilentVoid13** [comment]
Hey @tobulli this has already been implemented, I'm just waiting for the new obsidian API features to be released. Check #99

**Reply by tobulli** [reply]
Thanks for your reply! Looking forward to it :)
